### PR TITLE
Add scaffolding for Company

### PR DESCRIPTION
**Pull request by adrianMRJ**
# Description

- What?: We've added the scaffolding for companies, including models, controllers and views.
- Why?: These changes add the companies db for the admin to administrate

 # Testing

 - Just go to the route so you can see the scaffolds

### CODE REVIEW

File: app/assets/stylesheets/companies.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Companies controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline app/assets/stylesheets/companies.scss:3]: If you don't occupy this file, you can delete it. 

You can use this file to create your own styles related to your view.

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  margin: -7px -7px 0;
+  background-color: #c00;
+  color: #fff; }
+
+#error_explanation ul li {
+  font-size: 12px;
+  list-style: square; }
+
+label {
+  display: block; }
```
Comment (YilKanda) [inline app/assets/stylesheets/scaffolds.scss:65]: If you don't occupy this file, you can delete it.

File: app/helpers/companies_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module CompaniesHelper
+end
```
Comment (YilKanda) [inline app/helpers/companies_helper.rb:2]: If you don't occupy this file you can delete it.

File: app/controllers/companies_controller.rb [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  end
+
+  # DELETE /companies/1 or /companies/1.json
+  def destroy
+    @company.destroy
+    respond_to do |format|
+      format.html { redirect_to companies_url, notice: "Company was successfully destroyed." }
+      format.json { head :no_content }
+    end
+  end
```
Comment (YilKanda) [inline app/controllers/companies_controller.rb:57]: I recommend that they perform a logical deletion, that is, that the information in the database does not disappear, but is simply hidden from the end user.

File: spec/models/company_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Company, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (jclopezdev) [inline spec/models/company_spec.rb:4]: Please add some tests here

File: app/assets/stylesheets/companies.scss (hunk 1 above)
Comment (adrianMRJ) [reply app/assets/stylesheets/companies.scss]: Done

**Review by YilKanda — APPROVED** [review]
Look good to me! I would just like to recommend that you implement styles based on the stylesheets or the framework (like Bootstrap or Tailwind CSS)

**Comment by adrianMRJ** [comment]
We are using Tailwind, that's why we deleted the stylesheets generated with the scaffolds @YilKanda

**Pull request merged**